ra/ra.go

```diff
@@ -1050,19 +1050,35 @@ func (ra *RegistrationAuthorityImpl) issueCertificate(
 		RegistrationID: &acctIDInt,
 		OrderID:        &orderIDInt,
 	}
-	cert, err := ra.CA.IssueCertificate(ctx, issueReq)
-	if err != nil {
-		logEvent.Error = err.Error()
-		return emptyCert, err
-	}
 
-	if ra.ctpolicy != nil {
-		ra.getSCTs(ctx, cert.DER)
-	} else if ra.publisher != nil {
-		go func() {
-			// This context is limited by the gRPC timeout.
-			_ = ra.publisher.SubmitToCT(context.Background(), cert.DER)
-		}()
+	var cert core.Certificate
+	if features.Enabled(features.EmbedSCTs) {
+		precert, err := ra.CA.IssuePrecertificate(ctx, issueReq)
+		if err != nil {
+			logEvent.Error = err.Error()
+			return emptyCert, err
+		}
+		scts, err := ra.getSCTs(ctx, precert.DER)
+		if err != nil {
```

Comment:
Does this conditional body also need a `logEvent.Error = err.Error()` like the err checking from `ra.CA.IssuePrecertificate` and `ra.CA.IssueCertificate`?

Reply:
Done.